.gitignore

```diff
@@ -2,3 +2,4 @@ Locales/WeakAuras.lua
 Locales/WeakAuras_Options.lua
 Locales/WeakAuras_Tutorials.lua
 Locales/WeakAuras_Templates.lua
+.vscode/launch.json
```

Comment:
I've been working in VSCode, this just keeps the VSCode debugging settings file out of the repository.